pep-0674.rst

```diff
@@ -16,9 +16,9 @@ Incompatible C API change disallowing using macros as l-value to:
 * Ease the C API implementation on other Python implementation;
 * Help migrating existing C extensions to the HPy API.
 
-On the PyPI top 5000 projects, only 14 projects (0.3%) are affected by 4
-macro changes. Moreover, 24 projects just have to regenerate their
-Cython code to use ``Py_SET_TYPE()``.
+On the PyPI top 5000 projects, only 13 projects (0.3%) are affected by 2
+macro changes. Moreover, 22 projects just have to regenerate their
+Cython code.
```

Comment:
As a reader of the PEP, it confused me as to how there could be 22 projects that had to re-Cythonize, but only 13 that were affected. What I _think_ you mean is 

```suggestion
macro changes. An additional 22 projects just have to regenerate their
Cython code.
```

Reply:
Superceded by followup comment